Remarkable-Donkeys/bangazonCLI
# Popular Products and Revenue Report

# Steps To Test
1. git fetch --all
2. git checkout TB-1
3. cd src
4. dotnet run
5. check to make sure you have products, orders, and completed orders in your database.
6. ensure that popular products is displaying properly
7. ensure that the revenue report is displaying properly.
8. cd into  test
9. all tests should pass


https://github.com/Remarkable-Donkeys/bangazonCLI/issues/10
https://github.com/Remarkable-Donkeys/bangazonCLI/issues/11

### Description of what should happen when steps are followed
You should see a list of the top three popular products
You should see a revenue report with that customers revenue
All tests should pass

**At least one person needs to approve pull request before merged to master**
